tensorflow_datasets/image/the300w_lp.py

```diff
@@ -113,8 +114,19 @@ def _split_generators(self, dl_manager):
 
   def _generate_examples(self, image_dir_path):
     """Yields examples."""
-    image_files = tf.io.gfile.glob(
-        pattern=os.path.join(image_dir_path, "[!Code]*[!_Flip]/[!_]*.jpg"))
+    # PS C: \Users\VIJAY\Desktop\GitHub_Repos\datasets > conda list tensorflow
+    # packages in environment at C:\ProgramData\Miniconda3:
+    #
+    # Name                    Version                   Build  Channel
+    # tensorflow                2.1.0           eigen_py37hd727fc0_0
+    # tensorflow-base           2.1.0           eigen_py37h49b2757_0
+    # tensorflow-datasets       3.0.0-nightly             dev_0 < develop >
+    # tensorflow-estimator      2.1.0              pyhd54b08b_0
+    # tensorflow-io             0.12.0                   pypi_0    pypi
+    # tensorflow-metadata       0.21.2                   pypi_0    pypi
+    # PS C: \Users\VIJAY\Desktop\GitHub_Repos\datasets >
```

Comment:
I am not sure `tf.io.glob` error in TensorFlow for windows is resolved or not.

I will remove this file in future commits

Reply:
I think this has been solved in tf-nightly